pages/staking.jsx

```diff
@@ -587,46 +587,39 @@ const Staking = () => {
                     You have {tokenBalance} ergopad tokens.
                   </Typography>
                 )}
-                <Grid
-                  container
-                  spacing={3}
-                  alignItems="stretch"
-                  justifyContent="center"
-                  sx={{ flexGrow: 1 }}
-                >
-                  <Grid item md={10} xs={9}>
+                <Grid container sx={{mb: 2}} justifyContent={'space-between'} alignItems={'flex-start'}>
+                  <Grid item md={9} xs={9} sx={{ minWidth: 0}}>
                     <TextField
                       InputProps={{ disableUnderline: true }}
                       required
                       fullWidth
                       id="stakingAmount"
-                      label={`Enter the token amount you are staking`}
+                      label={`Token amount to stake`}
                       name="stakingAmount"
                       variant="filled"
-                      sx={{ mb: 2 }}
                       onChange={handleStakingFormChange}
                       value={stakingForm.tokenAmount}
                       error={stakingFormErrors.tokenAmount}
                       helperText={
-                        stakingFormErrors.tokenAmount &&
-                        'Enter a valid token amount'
+                        stakingFormErrors.tokenAmount && (tokenBalance ? 'Enter a valid token amount' : 'Please connect with yoroi or nautilus to use MaxAmount')
                       }
                     />
                   </Grid>
-                  <Grid item md={2} xs={3}>
-                    <Button
-                      onClick={() => {
-                        handleStakingFormChange({
-                          target: {
-                            name: 'stakingAmount',
-                            value: tokenBalance,
-                          },
-                        });
-                      }}
-                    >
-                      Max Amount
-                    </Button>
-                  </Grid>
+                    <Grid item md={2} xs={2} sx={{ mt: 0.5, minWidth: '63.03px', maxWidth: '63.03px'}}>
```

Comment:
Changed column count and changed width value to rem to allow for padding per comment
```suggestion
                    <Grid item md={2} xs={3} sx={{minWidth: '4.5rem', maxWidth: '4.5rem'}}>
```